backend/src/main/resources/graphql/admin.graphqls

```diff
@@ -5,6 +5,8 @@ extend type Query {
   organizations(identityVerified: Boolean): [Organization!]!
 }
 extend type Mutation {
+  uploadPatients(patientList: Upload!): String
+  resendToReportStream(testEventIds: [ID]!): Boolean
```

Comment:
not sure what would happend if you sent `[null]` to `findAllByInternalIdIn` but probably not good things. I think this type should be
```suggestion
  resendToReportStream(testEventIds: [ID!]!): Boolean
```